packages/noya-designsystem/src/components/GridView.tsx

```diff
@@ -64,37 +64,74 @@ const ItemDescription = styled.span(({ theme }) => ({
   textOverflow: 'ellipsis',
 }));
 
+type MenuItemType = 'delete' | 'copy';
+
 interface ItemProps {
-  children?: ReactNode;
-  title: string;
+  id?: string;
+  title?: string;
```

Comment:
Does `title` need to be optional?

Reply:
I think not, I changed it for other reason. I'll keep it as it was now.